src/test/java/uk/gov/companieshouse/api/accounts/controller/FilingControllerTest.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
Where not using mockito yet? Is it worth removing until we do or did you add it to save adding it in future?